src/main/java/seedu/address/model/tag/TagContainsKeywordsPredicate.java

```diff
@@ -0,0 +1,45 @@
+package seedu.address.model.tag;
+
+import java.util.Set;
+import java.util.function.Predicate;
+
+import seedu.address.model.person.Person;
+
+
+/**
+ * Tests that a {@code Person}'s {@code Tag} matches any of the tags given.
+ */
+public class TagContainsKeywordsPredicate implements Predicate<Person> {
```

Comment:
Everything here LGTM! I suggest you could create test for this file to improve the diff coverage